datashuttle/utils/ds_logger.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from pathlib import Path
+from typing import Any, List, Optional
+
+import fancylog as package
+from fancylog import fancylog
+from rich import print as rich_print
+from rich.console import Console
+from rich.filesize import decimal
+from rich.markup import escape
+from rich.text import Text
+from rich.tree import Tree
+
+from . import utils
+
+
+def start(
+    path_to_log: Path, name: str, variables: Optional[List[Any]]
+) -> None:
+    """"""
+    fancylog.start_logging(
+        path_to_log,
+        package,
+        filename=name,
+        variables=variables,
+        verbose=False,
+        timestamp=True,
+        file_log_level="INFO",
```

Comment:
Is there a reason for setting this as "INFO"? I think this means that "DEBUG" information (which should be the majority) wouldn't be saved to the log.

Reply:
I think the only reason was that I did not understand the logging levels sufficiently, will edit this!